src/frontend/src/common/components/library/data_subview/components/DownloadModal/index.tsx

```diff
@@ -155,12 +155,7 @@ const DownloadModal = ({
         </B>{" "}
         <ContactUsLink />
       </Notification>
-      <Dialog
-        disableEscapeKeyDown
-        disableBackdropClick
-        open={open}
-        onClose={handleCloseModal}
-      >
+      <Dialog disableEscapeKeyDown open={open} onClose={handleCloseModal}>
```

Comment:
why was `disableBackdropClick` removed?

Reply:
The prop was removed as part of the migration ([docs](https://mui.com/material-ui/migration/v5-component-changes/#%E2%9C%85-remove-disablebackdropclick-prop)).  They recommend handling this in onClose if we want to.  Do you know why we're overriding the default here?

Reply:
I will either remove the prop from our BaseDialog component or add it to onClose there

Reply:
I have no idea why.